c/include/libsbp/file_io.h

```diff
@@ -74,35 +72,31 @@ typedef struct SBP_ATTR_PACKED {
 
 /** List files in a directory (host => device)
  *
- * The read directory message lists the files in a directory on the
- * device's onboard flash file system.  The offset parameter can be
- * used to skip the first n elements of the file list. Returns a
- * MSG_FILEIO_READ_DIR_RESP message containing the directory
- * listings as a NULL delimited list. The listing is chunked over
- * multiple SBP packets. The sequence number in the request will be
- * returned in the response.  If message is invalid, a followup
- * MSG_PRINT message will print "Invalid fileio read message".
- * A device will only respond to this message when it is received
- * from sender ID 0x42.
+ * The read directory message lists the files in a directory on the device's
+ * onboard flash file system.  The offset parameter can be used to skip the first
+ * n elements of the file list. Returns a MSG_FILEIO_READ_DIR_RESP message
+ * containing the directory listings as a NULL delimited list. The listing is
+ * chunked over multiple SBP packets. The sequence number in the request will be
+ * returned in the response.  If message is invalid, a followup MSG_PRINT message
+ * will print "Invalid fileio read message". A device will only respond to this
+ * message when it is received from sender ID 0x42.
  */
 #define SBP_MSG_FILEIO_READ_DIR_REQ  0x00A9
 
 typedef struct SBP_ATTR_PACKED {
   u32 sequence;    /**< Read sequence number */
-  u32 offset;      /**< The offset to skip the first n elements of the file list
- */
+  u32 offset;      /**< The offset to skip the first n elements of the file list */
   char dirname[0];  /**< Name of the directory to list */
 } msg_fileio_read_dir_req_t;
 
 
 /** Files listed in a directory (host <= device)
  *
- * The read directory message lists the files in a directory on the
- * device's onboard flash file system. Message contains the directory
- * listings as a NULL delimited list. The listing is chunked over
- * multiple SBP packets and the end of the list is identified by an
- * entry containing just the character 0xFF. The sequence number in
- * the response is preserved from the request.
+ * The read directory message lists the files in a directory on the device's
```

Comment:
I'm happy to reduce the width of the `wordwrap` so that this produces less noise in the diff, or if the coding standard requires a fewer CPL. There will still be differences, but should be fewer.